lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/ProcessCriCallbackHandler.java

```diff
@@ -154,8 +154,11 @@ public APIGatewayProxyResponseEvent handleRequest(
                     var criOAuthSessionItem =
                             criOAuthSessionService.getCriOauthSessionItem(
                                     callbackRequest.getState());
-                    pageOutput.put(
-                            "clientOAuthSessionId", criOAuthSessionItem.getClientOAuthSessionId());
+                    if (criOAuthSessionItem != null) {
+                        pageOutput.put(
+                                "clientOAuthSessionId",
+                                criOAuthSessionItem.getClientOAuthSessionId());
+                    }
```

Comment:
Although we know the `clientOAuthSessionId` exists to get to this point, we can't guarantee that the corresponding session still does (it may have expired).